src/components/Button.tsx

```diff
@@ -2,29 +2,41 @@ import * as React from 'react'
 import styled from '@emotion/styled'
 
 import { getEmSize } from '../styles/mixins'
+import { colors } from '../styles/variables'
 import { Link, StaticQuery, graphql } from 'gatsby'
 
 const StyledButton = styled.div`
+  font-size: 20px;
   display: inline-flex;
-  padding: ${getEmSize(6)}em;
   cursor: pointer;
-  border-radius: ${getEmSize(6)}em;
-  background-color: rgb(255, 229, 0);
-  box-shadow: rgba(0, 0, 0, 0.5) 1px 4px 13px 0px;
+  border-radius: 1000px;
```

Comment:
You can instead use 50%, which should be more clear

Reply:
> You can instead use 50%, which should be more clear

Sadly this doesn't work. Using a big border-radius is the workaround pretty much everyone uses